conda/recipes/cudf_kafka/meta.yaml

```
@@ -28,12 +28,11 @@ requirements:
   host:
     - python
     - cython >=0.29,<0.30
-    - setuptools
     - cudf {{ version }}
     - libcudf_kafka {{ version }}
   run:
     - libcudf_kafka {{ version }}
```

Appears we are missing `python` from `run` here

```suggestion
    - python
    - libcudf_kafka {{ version }}
```